Sources/Public/CombineOperatorOption.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CombineOperatorOption.swift
+//  SwiftPrettyPrint
+//
+//  Created by Yusuke Hosonuma on 2020/11/26.
+//
+
+// Namespace
+public enum CombineOperatorOption {}
+
+extension CombineOperatorOption {
+    public enum Format {
+        case singleline
+        case multiline
+    }
+
+    public enum When: CaseIterable {
```

Comment:
nit: 
I prefer `Event` to `When`
How do you think about it.

Reply:
I was at a loss too.
I'll change it!

Reply:
fixed: d4d90a3